src/main/java/org/broadinstitute/hellbender/utils/MathUtils.java

```diff
@@ -1480,4 +1484,41 @@ public static boolean allMatch(final int[] array, final IntPredicate pred) {
         }
         return true;
     }
+
+    /**
+     *
+     * @param array array of integers
+     * @return index of the max. In case of a tie, return the smallest index
+     */
+    public static int maxElementIndex(final int[] array){
+        int maxIndex = 0;
+        int currentMax = Integer.MIN_VALUE;
+        for (int i = 0; i < array.length; i++){
+            if (array[i] > currentMax){
+                maxIndex = i;
+                currentMax = array[i];
+            }
+        }
+        return maxIndex;
+    }
+
+    /**
+     *
+     * Computes the log10 probability density of BetaBinomial(k|n, alpha, beta)
+     *
+     * @param alpha pseudocount of number of heads
+     * @param beta pseudocount of number of tails
+     * @param k value to evaluate
+     * @param n number of coin flips
+     * @return probability density function evaluated at k
+     */
+    public static double log10BetaBinomialProbability(final int k, final int n, final double alpha, final double beta){
```

Comment:
In light of the last comment, just checking that this is intentional code for this PR and not a rebasing artifact.

Reply:
Yup all is well I believe